index.html

```diff
@@ -0,0 +1,396 @@
+<!DOCTYPE html>
+<html class="no-skrollr">
+    <head>
+        <title>
+            Let's help ourselves
+        </title>
+        <link href="assets/images/globe.png" rel="icon" sizes="16x16" type="image/gif"/>
+        <link href="css/styles.css" rel="stylesheet" type="text/css">
+        </link>
+        <script src="assets/lib/jquery-1.11.3.min.js"></script>
+        <script src="assets/lib/skrollr.min.js" type="text/javascript"></script>
+        <script src="assets/lib/jquery.scrollTo.min.js" type="text/javascript"></script>
+        <script src="assets/lib/jquery-sectionsnap.js" type="text/javascript"></script>
+        <script src="assets/lib/smoothscroll.js" type="text/javascript"></script>
+        <script src="js/script.js" type="text/javascript"></script>
+    </head>
+    <body>
+        <header>
+            <div id="header">
+                <img src="assets/images/header_right.png">
+                    <div id="header_contatiner">
+                        <div id="site_title">
+                            Let's Help Ourselves
+                        </div>
+                    </div>
+                </img>
+            </div>
+        </header>
+        <div class="sections-wrapper">
```

Comment:
Use consistent naming convention for class/id. Underscored class names are being used throughout the page. Please change class name accordingly.